### DISCUSSION TITLE
Επεξεργασία markdown

### DISCUSSION THREAD

**Discussion by mxidakis** [sdy60/syllabus#160] [Q&A] [answered]
Θα μπορούσατε να προτείνετε ένα πρόγραμμα για την επεξεργασία των αρχείων *.md ;

**Comment by Pagiatis-Spiros** [comment, marked as answer]
Δημιούργησε το readme.md στο repository που έκανες και την επεξεργασία την κάνεις απευθείας εκεί (πατώντας το μολυβάκι πάνω δεξιά στο πλαίσιο κάνεις επεξεργασία).

**Reply by epidrome** [reply]
Μπορεί να γίνει και με εφαρμογή απλού επεξεργαστή κειμένου ή με κάποιον εξειδικευμένο για να υπάρχει προεπισκόπηση. Η πιο απλή λύση όμως για την περίπτωση της ΓΕ είναι πράγματι ο επεξεργαστής κειμένου εδώ που έχει και προεπισκόπηση.

**Reply by Pagiatis-Spiros** [reply]
Σας ευχαριστούμε για την επιβεβαίωση.
